internal/operators/cnv/cnv_operator.go

```diff
@@ -197,20 +199,45 @@ func (o *operator) getDevicesMemoryOverhead(host *models.Host) (int64, error) {
 		return 0, err
 	}
 	gpuCount := o.getGPUCount(*inventory)
-	// One GPU imposes 1GiB of additional memory requirement
-	return conversions.GibToMib(gpuCount), nil
+	srIovNicCount := o.getSrIovNicCount(*inventory)
+	totalDevices := gpuCount + srIovNicCount
+
+	// One device imposes 1GiB of additional memory requirement
+	return conversions.GibToMib(totalDevices), nil
 }
 
 func (o *operator) getGPUCount(inventory models.Inventory) int64 {
 	var gpuCount int64
 	for _, gpu := range inventory.Gpus {
-		if o.config.SupportedGPUs[getDeviceKey(gpu)] {
-			gpuCount = gpuCount + 1
+		if o.config.SupportedGPUs[getDeviceKeyForGPU(gpu)] {
+			gpuCount++
 		}
 	}
 	return gpuCount
 }
 
-func getDeviceKey(gpu *models.Gpu) string {
-	return gpu.VendorID + ":" + gpu.DeviceID
+func (o *operator) getSrIovNicCount(inventory models.Inventory) int64 {
+	var srIovCount int64
```

Comment:
int64? How many nics are you expecting :laughing:

Reply:
FYI you don't need to change this.

Reply:
:D It's `int64` to avoid casting the result.